### DISCUSSION TITLE
Errors on empty fields (BT-5101(a)-Lot, BT-5071-Lot, BT-5141-Lot, BT-161-NoticeResult, BT-119-LotResult)

### DISCUSSION THREAD

**Discussion by edatelli** [OP-TED/eForms-SDK#967] [Q&A] [answered]
We are trying to publish a notice but can't fix the errors on these empty fields. Is it possible that even though we have emptied the fields, the system doesn't clear the cache or cookies or data storage/database? I can provide the ID for the contract award notice we are trying to publish too if needed.
![image](https://github.com/OP-TED/eForms-SDK/assets/174586610/a57baee0-6be9-48a7-b175-c8855530c0bb)
![image](https://github.com/OP-TED/eForms-SDK/assets/174586610/7c301700-acfd-4fc6-a1e5-2bff40a27000)
![image](https://github.com/OP-TED/eForms-SDK/assets/174586610/455a4282-a13e-47c2-97ea-14c5c64f5010)

**Comment by YvesJo** [comment, marked as answer]
Hi,
This question is not SDK related, I invite you to get in touch with the Helpdesk.
KR
